docker/install-snp-dists.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
I think install scripts should be in a directory called `install/` . Another little thing I thought could be improved in btb-seq.